modules/runtime/nuxeo-stream/src/test/java/org/nuxeo/lib/stream/tests/log/TestLogChronicle.java

```diff
@@ -74,6 +68,8 @@ public LogManager createManager() throws Exception {
 
     @Test
     public void deleteInvalidPath() throws Exception {
+        org.junit.Assume.assumeFalse(IS_WIN);
```

Comment:
the reason to exclude test under windows was that temp folder are not cleaned, 
you need to provide a reason for each assumeFalse now
assumeFalse("explain why or NXP", IS_WIN)